### PR TITLE
Add further description to VM settings

### PR DESCRIPTION
**Pull request by michaellihs**
Extended list of VM settings if you run into the problem of booting into a black screen when starting the VM. I had this issue with your original tutorial and found some settings that might be helpful for others as well to prevent those problems :-)

Thanks a lot for your work!

### CODE REVIEW

**Comment by geerlingguy** [comment]
With the numbering changes, it's hard to see what's different in GitHub's UI... what are the main differences in this PR?

**Comment by michaellihs** [comment]
Thanks for your fast response!

I reverted the number changes and chose a format that makes differences clearer. Main differences is the check list with VM settings to prevent VirtualBox not booting the ISO image. Found those settings here http://webcache.googleusercontent.com/search?q=cache:-HetrOnSp5sJ:appledigg.com/t/install-mac-os-x-el-capitan-yosemite-in-virtualbox-in-mac-os-x-tutorial/147+&cd=1&hl=en&ct=clnk&gl=de (non-cached page is no longer available as it seems...).

File: README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ To build a VM running Mac OS X, follow the directions below:
   4. Set name, type, and version (set to Mavericks for 'Version').  (This might not be necessary in the latest VirtualBox release.)
   5. Follow the rest of the VM creation wizard and either leave the defaults or adjust to your liking.
   6. In Terminal, run the command `VBoxManage modifyvm "{vmname}" --cpuidset 00000001 000306a9 00020800 80000201 178bfbff` (where "Mac OS X" is the exact name of the OS X VM set in step 4) so the VM has the right CPU settings for macOS.
+    * When VirtualBox boots into a "black screen", check the following properties in the VM settings
```
Comment (geerlingguy) [inline README.md]: Should this be 'If'? Unless it happens every time (I haven't built a VM for Mac OS X in a few months).

Comment (michaellihs) [inline README.md]: you are right - my bad... english isn't my native language :-) Fixed it and made a new commit...

**Comment by mi-hol** [comment]
@geerlingguy @michaellihs is this PR still valid or should it be closed?

**Comment by marco-m** [comment]
@geerlingguy @michaellihs is this PR still valid or should it be closed?

**Comment by geerlingguy** [comment]
Yeah I think it can be closed for now.

**Pull request closed without merging**